sound/soc/sof/ops.h

```diff
@@ -474,12 +474,12 @@ snd_sof_pcm_platform_trigger(struct snd_sof_dev *sdev,
 	return 0;
 }
 
-/* Firmware loading */
-static inline int snd_sof_load_firmware(struct snd_sof_dev *sdev)
+/* Firmware loading. fw_filename must contain the full path to the file */
```

Comment:
I see, but then it should also say that it can be left to NULL to use the strings from the default profile.